8-application-demos/6-kalshi-bet-predictor/main.py

```diff
@@ -0,0 +1,170 @@
+import torch
+from transformers import AutoModelForCausalLM, AutoTokenizer
+from typing import Tuple
+import requests
+import re
+from dotenv import load_dotenv
+import os
+from exa_py import Exa
+
+def getKalshiQuestion(market_ticker)->Tuple[str,str]:
```

Comment:
why make 2 requests to this endpoint? Make 1 and just store the results to be used